omnibus/config/software/datadog-agent-integrations.rb

```diff
@@ -167,11 +170,34 @@
       end
     end
 
+    if windows?
+      static_reqs_file = "#{windows_safe_path(install_dir)}\\#{agent_requirements_file}"
+    else
+      static_reqs_file = "#{install_dir}/#{agent_requirements_file}"
+
+    # Remove any blacklisted requirements from the static-environment req file
+    tmp = []
+    File.open("#{static_reqs_file}", 'r+') do |f|
```

Comment:
To read all the lines of the file you can simply use `File.open("#{static_reqs_file}", 'r+').readlines()` here (see https://ruby-doc.org/core-2.1.0/IO.html#method-i-readlines)

Reply:
Nice, was looking for that when working on this PR 😄thanks! As you can tell, I don't often Ruby.

Reply:
> I don't often Ruby

Don't say it like it's a bad thing ;)